<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/EditorResources.qrc">
    <normaloff>:/editor/hammergrau.png</normaloff>:/editor/hammergrau.png</iconset>
  </property>
  <widget class="QWidget" name="mdiArea">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Level"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_Terrain"/>
    <addaction name="actionEdit_Objects"/>
    <addaction name="separator"/>
    <addaction name="actionSave_grouped_Meshes"/>
    <addaction name="actionSave_grouped_Objects"/>
    <addaction name="actionSave_Collision"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionEditor_Settings"/>
    <addaction name="actionLevel_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionSnap_on_Grid"/>
    <addaction name="actionSnap_on_Angle"/>
    <addaction name="actionSnap_on_Scale"/>
    <addaction name="actionPhysics"/>
    <addaction name="actionFog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Level"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionMove_Mode"/>
   <addaction name="actionRotate_Mode"/>
   <addaction name="actionScale_Mode"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew_Level">
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/New.png</normaloff>:/editor/New.png</iconset>
   </property>
   <property name="text">
    <string>New Level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/Open.png</normaloff>:/editor/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/Save.png</normaloff>:/editor/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEdit_Terrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit Terrain</string>
   </property>
  </action>
  <action name="actionEdit_Objects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit Objects</string>
   </property>
  </action>
  <action name="actionSave_grouped_Meshes">
   <property name="text">
    <string>Save grouped Meshes</string>
   </property>
  </action>
  <action name="actionSave_grouped_Objects">
   <property name="text">
    <string>Save grouped Objects</string>
   </property>
  </action>
  <action name="actionSave_Collision">
   <property name="text">
    <string>Save Collision</string>
   </property>
  </action>
  <action name="actionEditor_Settings">
   <property name="text">
    <string>Editor Settings</string>
   </property>
  </action>
  <action name="actionLevel_Settings">
   <property name="text">
    <string>Level Settings</string>
   </property>
  </action>
  <action name="actionSnap_on_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap on Grid</string>
   </property>
  </action>
  <action name="actionSnap_on_Angle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap on Angle</string>
   </property>
  </action>
  <action name="actionSnap_on_Scale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Snap on Scale</string>
   </property>
  </action>
  <action name="actionPhysics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Physics</string>
   </property>
  </action>
  <action name="actionFog">
   <property name="text">
    <string>Fog</string>
   </property>
  </action>
  <action name="actionMove_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/arrow_move.png</normaloff>:/editor/arrow_move.png</iconset>
   </property>
   <property name="text">
    <string>Set Move Mode</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionRotate_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/arrow_rotate.png</normaloff>:/editor/arrow_rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate Mode</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionScale_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/EditorResources.qrc">
     <normaloff>:/editor/arrow_scale.png</normaloff>:/editor/arrow_scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale Mode</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/EditorResources.qrc"/>
 </resources>
 <connections/>
</ui>
